### DISCUSSION TITLE
получение позиции игрока

### DISCUSSION THREAD

**Discussion by justqstn** [Sokol-Games-GmbH/PixelCombats.PublicApi#87] [Ideas]
Index GetPosition { get; } // возвращает координаты игрока
void SetPosition(index) // перемещение игрока по координатам

еще предлагаю другой вариант: 
Index Position { get; set; } // текущие координаты игрока

**Comment by forwork10** [comment]
Очень полезная вещь, также стоит добавить возможность изменить направление камеры игрока.

**Comment by kkohno** [comment]
Взяли на заметку, сделаем

**Comment by kkohno** [comment]
сделано, ждите 510 версию
